[Sentry] ModuleIntegration wasn't able to extract modules: module integration failed

<!-- If you suspect this could be a bug, follow the template. -->

### What version of Dgraph are you using?
Master

### Have you tried reproducing the issue with the latest release?
Yes

### What is the hardware spec (RAM, OS)?
24GB Memory and Ubuntu 18.04.4 LTS

### Steps to reproduce the issue (command/config used to run Dgraph).
Ctrl-c on my Dgraph zero instance and received the following error.

![image](https://user-images.githubusercontent.com/20114263/79001111-eb493b80-7b1b-11ea-9208-883a8911e65a.png)

```
^CI0410 10:55:42.313708    1256 run.go:266] --- Received interrupt signal
I0410 10:55:42.314014    1256 run.go:285] Shutting down...
E0410 10:55:42.314227    1256 http.go:252] Stopped taking more http(s) requests. Err: accept tcp [:
:]:6080: use of closed network connection
panic: send on closed channel

goroutine 228 [running]:
github.com/dgraph-io/dgraph/raftwal.(*DiskStorage).CreateSnapshot(0xc0064eaf80, 0x21e79a, 0xc00044e
b58, 0xc000300840, 0x15f, 0x15f, 0x0, 0x0)
        /home/dmitry/dgraph/raftwal/storage.go:692 +0x368
github.com/dgraph-io/dgraph/dgraph/cmd/zero.(*node).trySnapshot(0xc0000f3680, 0x0)
        /home/dmitry/dgraph/dgraph/cmd/zero/raft.go:614 +0x1c9
github.com/dgraph-io/dgraph/dgraph/cmd/zero.run.func3(0xc000496ac0, 0xc00734e180, 0x1c2d000, 0xc000
0f4060, 0xc0064eaf80, 0x1c2d000, 0xc0000f4040)
        /home/dmitry/dgraph/dgraph/cmd/zero/run.go:297 +0x1b7
created by github.com/dgraph-io/dgraph/dgraph/cmd/zero.run
        /home/dmitry/dgraph/dgraph/cmd/zero/run.go:282 +0xf70
[Sentry] 2020/04/10 10:55:42 ModuleIntegration wasn't able to extract modules: module integration f
ailed
[Sentry] 2020/04/10 10:55:42 Sending fatal event [cc19eb68a1de4cdc8f4808283e9d17ea] to sentry.io pr
oject: 1805390
[Sentry] 2020/04/10 10:55:42 Buffer flushed successfully.
```

### Expected behaviour and actual result.

Sentry should not output this error? :D

--- a/go.mod
+++ b/go.mod
@@ -23,7 +23,7 @@ require (
 	github.com/dgryski/go-farm v0.0.0-20191112170834-c2139c5d712b
 	github.com/dgryski/go-groupvarint v0.0.0-20190318181831-5ce5df8ca4e1
 	github.com/dustin/go-humanize v1.0.0
-	github.com/getsentry/sentry-go v0.5.1
+	github.com/getsentry/sentry-go v0.6.0
 	github.com/go-ini/ini v1.39.0 // indirect
 	github.com/go-sql-driver/mysql v0.0.0-20190330032241-c0f6b444ad8f
 	github.com/gogo/protobuf v1.3.1

